src/models/model_DatasetFullBreakdownsBreakdownValuesInner.jl

```diff
@@ -2,15 +2,36 @@
 # Do not modify this file directly. Modify the OpenAPI specification instead.
 
 
-@doc raw"""DatasetFull_breakdowns_breakdownValues_inner
-
-    DatasetFullBreakdownsBreakdownValuesInner(;
-        name=nothing,
-        description=nothing,
-    )
+"""
+```
+DatasetFullBreakdownsBreakdownValuesInner(;
+    name=nothing,
+    description=nothing,
+)
+```
+The fuction give the name and the description of the dataset.
```

Comment:
Could you adapt my comments from the `DataTableFullVariablesInner` file for here?